docs/openshift/index.rst

```diff
@@ -95,21 +95,44 @@ The table below shows what BIG-IP configurations the |kctlr| applies for common
                                    BIG-IP system.
    ============================  ==========================================================
 
+Advanced Deployments
+````````````````````
+
+The |octlr-long| supports the following OpenShift `Advanced Deployment Strategies`_:
+
+- `Blue-Green Deployment`_
+- `A/B Deployment`_
+
+Follow the instructions provided in the OpenShift documentation to use these deployment strategies with your BIG-IP Controller and BIG-IP device(s).
+
+.. important::
+
+   The |octlr-long| provides the following advantages over the native HAProxy when working with `alternate backends`_:
+
+   - You use any BIG-IP-supported load balancing algorithm, not just round robin. [#lb]_
```

Comment:
maybe 'any BIG-IP load balancing algorithm supported by the BIG-IP controller, not just round robin.'